src/main/java/com/example/HelloWorldSpring/HelloWorldSpringController.java

```diff
@@ -1,12 +1,19 @@
 package com.example.HelloWorldSpring;
 
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class HelloWorldSpringController {
     @GetMapping("/hello")
-    public String hello(){
-        return "Hello World!";
+    public String hello(@RequestParam(value = "country", defaultValue = " ") String country){
```

Comment:
空文字を表現したいならこのようにしたほうがいいです。
```suggestion
    public String hello(@RequestParam(value = "country", defaultValue = "") String country){
```

Reply:
空文字部分修正しました。